docs/generate_metadata.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
```

Comment:
I'm not sure we need this file at all. As far as I know it's needed for the internal doc server. But I guess we want to upload to PyPI?

Reply:
Yes you are right! But from what I understood, we also need to upload the repository to our Software Catalog. This step is requiring the creation of `metadata.md` containing the structure found in this file. I can maybe double check with someone having experience with open-sourcing steps.

Reply:
I see. Personally I don't know about the policies on that. I'm actually wondering how the internal upload would be done since in the past it was done from Jenkins/GitLab CI, which had access to intranet, but I'm not sure this would work on GitHub.

Anyway, I think you know more than me about this :)

Reply:
I am not sure about everything concerning the policy! I will try to find out and let you know!

Reply:
I temporarily removed the file. I will discuss the metadata generation and open a new PR if this file is needed. So maybe we can merge this PR! What do you think ?

Reply:
awesome, sounds good :)